blogs/2024/11/11/introducing-github-copilot-for-azure.md

```diff
@@ -0,0 +1,78 @@
+
+I'm thrilled to introduce the public preview of **GitHub Copilot for Azure** - a new tool that integrates effortlessly with GitHub Copilot Chat in VS Code. Imagine it as your personal guide for navigating the Azure cloud. No more toggling between your IDE and the Azure portal to manage infrastructure or look up commands and arguments. Now, you can concentrate on your core task - coding. Whether you're setting up services or deploying applications, simply ask in GitHub Copilot Chat and manage everything directly within your editor.
```

Comment:
Can you add a line with the date and you name and title? Something like:

```
November 12, 2024 by Chris Harris, [your title and optional social link]
```

Reply:
Done